src/AvtaleSide/steg/BeregningTilskudd/BeregningTilskuddSteg.tsx

```diff
@@ -31,7 +41,20 @@ const BeregningTilskuddSteg = (props: Context) => {
     return (
         <Innholdsboks utfyller="veileder_og_arbeidsgiver">
             <SkjemaTittel>Beregning av lønnstilskudd</SkjemaTittel>
-            <Undertittel>Lønn og stillingsprosent</Undertittel>
+            <Undertittel>Lønnstilskudd prosent</Undertittel>
+            <Normaltekst className="beregningTilskuddSteg__luft">
```

Comment:
Her kan det brukes BEMHelper, sånn her: const cls = BEMHelper('beregningTilskuddSteg'); og så: `className={cls.element('luft')}>`

Reply:
BeregningTilskudSteg trenger en del design sjekking, den var ikke hovedfokus, da hovedfokus var utregningen for oppgaven, men jeg kan prøve å forbedre den litt.